pilot/pkg/model/sidecar.go

```diff
@@ -105,7 +105,7 @@ type SidecarScope struct {
 	// This is the namespace where the sidecar takes effect,
 	// maybe different from the ns where sidecar resides if sidecar is in root ns.
 	Namespace string
-	// The crd itself. Can be nil if we are constructing the default
+	// The cr itself. Can be nil if we are constructing the default
```

Comment:
nit: Is this intentional?

Reply:
cr short for custom resource, so I think it's intentional and more right.